web/src/app/services/survey/survey.service.ts

```diff
@@ -61,6 +61,10 @@ export class SurveyService {
     this.activeSurvey$.subscribe(survey => (this.currentSurvey = survey));
   }
 
+  getCurrentSurvey(): Survey {
```

Comment:
Nit: I think active vs current isn't unclear a first glance. Really this is exposing the non-reactive version of `activeSurvey$`. Maybe just expose it as a readonly property called `activeSurvey`, perhaps a la https://stackoverflow.com/a/74165349/9986466 ? Same throughout.

Reply:
Done.